frontend/src/pages/style/Identification.scss

```diff
@@ -0,0 +1,53 @@
+:root {
```

Comment:
Don't initialize your variables anywhere else than `index.scss` : here, you'll have these values overwrite the original ones, but ONLY if you load this component, it'd be a hard-to-understand bug

Reply:
done ;)